pkg/common/aws/utils.go

```diff
@@ -131,3 +139,15 @@ func createCcsKeys() error {
 	}
 	return err
 }
+
+func populateAwsKeys() {
+	//If provider is Rosa, grab the keys
+	if viper.GetString(config.Provider) == "rosa" {
+		accessKey = viper.GetString("rosa.awsAccessKey")
+		secretAcccessKey = viper.GetString("rosa.awsSecretAccessKey")
+	} else {
+		//If provider is AWS, grab the keys from the ccsKeys variable
+		accessKey = viper.GetString("ocm.aws.accessKey")
+		secretAcccessKey = viper.GetString("ocm.aws.secretKey")
```

Comment:
```suggestion
		accessKey = viper.GetString(rosaprovider.AWSAccessKeyID)
		secretAcccessKey = viper.GetString(rosaprovider.AWSSecretAccessKey)
	} else {
		//If provider is AWS, grab the keys from the ccsKeys variable
		accessKey = viper.GetString(ocmprovider.AWSAccessKey)
		secretAcccessKey = viper.GetString(ocmprovider.AWSSecretKey)
```

we can use the constants here